hoot-core/src/main/cpp/hoot/core/conflate/poi-polygon/PoiPolygonSchema.cpp

```diff
@@ -207,31 +207,17 @@ bool PoiPolygonSchema::hasMoreThanOneType(const ConstElementPtr& element)
 {
   int typeCount = 0;
   QStringList typesParsed;
-  if (_allTagKeys.size() == 0)
+  if (_allTypeKeys.size() == 0)
```

Comment:
This basically gives you all schema tag keys minus any that aren't associated with an element's type.